### PR TITLE
Pop-Up Modal

### PR DESCRIPTION
**Pull request by mprotic123**
Pull request created in order to review the project "Pop-Up modal" from Microverse.

### CODE REVIEW

File: personal-portfolio/js/localstorage.js [hunk 1]
```diff
@@ -1,37 +1,72 @@
...
+  inputEmail.value += localStorage.getItem('email-address');
+  inputMessage.value += localStorage.getItem('message');
+}
+
+if (localStorage.getItem('full-name')) {
+  preFillData();
+} else {
+  storageLocally();
+}
+
```
Comment (carlos-ssh) [inline personal-portfolio/js/localstorage.js:32]: ## Your projects should be in an array of objects. Kindly work on it.

Screen Shot 2021-10-06 at 11 07 18

## Fix your linter issues instead disable many rules:
Screen Shot 2021-10-06 at 11 09 26

File: personal-portfolio/js/localstorage.js [hunk 2]
```diff
@@ -1,37 +1,72 @@
...
-inputName.onchange = storageLocally;
-inputEmail.onchange = storageLocally;
-inputMessage.onchange = storageLocally;
+
+const inputName = document.getElementById('full-name');
+const inputLastName = document.getElementById('Last-name');
+const inputEmail = document.getElementById('email-address');
+const inputMessage = document.getElementById('message');
+const formLocally = document.getElementById('form');
+
```
Comment (carlos-ssh) [inline personal-portfolio/js/localstorage.js:7]: ## Kindly fix your design issue:
- This container is overlapping ``x-axis``.
Screen Shot 2021-10-06 at 11 05 41

File: personal-portfolio/js/localstorage.js [hunk 3]
```diff
@@ -1,37 +1,72 @@
...
+  localStorage.setItem('message', localMessage);
+}
+
+function preFillData () {
+  inputName.value += localStorage.getItem('full-name');
+  inputLastName.value += localStorage.getItem('Last-name');
+  inputEmail.value += localStorage.getItem('email-address');
+  inputMessage.value += localStorage.getItem('message');
+}
+
```
Comment (carlos-ssh) [inline personal-portfolio/js/localstorage.js:26]: Provide a link to your License in your ``README`` file.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @mprotic123 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: personal-portfolio/css/style.css [hunk 4]
```diff
@@ -750,14 +750,17 @@ button.btn-1:hover {
```
Comment (SajjadAhmad14) [inline personal-portfolio/css/style.css:750]: Pop-up still does not follow the design on both desktop and mobile. `See live` and `See Source` buttons should be on the right side of the project title. The mobile view does not show these buttons. Kindly make the necessary changes while following the design exactly
btns
.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Milan Protić, 
 ### Thank you for working on the changes suggested. :ok_hand:

## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: personal-portfolio/css/style.css [hunk 5]
```diff
@@ -750,14 +750,17 @@ button.btn-1:hover {
 
 #icon-1 {
```
Comment (SajjadAhmad14) [inline personal-portfolio/css/style.css:751]: [ OPTIONAL ] Please add an icon to the `See source` button on the mobile view
optional

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Milan Protić, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
# OPTIONAL BUT RECOMMENDED
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by mprotic123** [comment]
Pull request approved.

**Pull request closed without merging**